saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -231,76 +349,43 @@ def test_order_lines_have_all_required_fields(order, order_line_with_one_allocat
-    unit_net_amount = line.unit_price_net_amount.quantize(Decimal("0.01"))
-    unit_gross_amount = line.unit_price_gross_amount.quantize(Decimal("0.01"))
     unit_discount_amount = line.unit_discount_amount.quantize(Decimal("0.01"))
+    line_payload = lines_payload[0]
     allocation = line.allocations.first()
-    undiscounted_unit_price_net_amount = (
-        line.undiscounted_unit_price.net.amount.quantize(Decimal("0.01"))
-    )
-    undiscounted_unit_price_gross_amount = (
-        line.undiscounted_unit_price.gross.amount.quantize(Decimal("0.01"))
-    )
-    undiscounted_total_price_net_amount = (
-        line.undiscounted_total_price.net.amount.quantize(Decimal("0.01"))
-    )
-    undiscounted_total_price_gross_amount = (
-        line.undiscounted_total_price.gross.amount.quantize(Decimal("0.01"))
-    )
-
-    total_line = line.total_price
     global_warehouse_id = graphene.Node.to_global_id(
         "Warehouse", allocation.stock.warehouse_id
     )
     product = line.variant.product
     product_type = product.product_type
-    assert line_payload == {
-        "id": line_id,
-        "type": "OrderLine",
-        "product_name": line.product_name,
-        "variant_name": line.variant_name,
-        "translated_product_name": line.translated_product_name,
-        "translated_variant_name": line.translated_variant_name,
-        "product_sku": line.product_sku,
-        "product_variant_id": line.product_variant_id,
-        "quantity": line.quantity,
-        "currency": line.currency,
-        "unit_discount_amount": str(unit_discount_amount),
-        "unit_discount_type": line.unit_discount_type,
-        "unit_discount_reason": line.unit_discount_reason,
-        "unit_price_net_amount": str(unit_net_amount),
-        "unit_price_gross_amount": str(unit_gross_amount),
-        "total_price_net_amount": str(total_line.net.amount.quantize(Decimal("0.01"))),
-        "total_price_gross_amount": str(
-            total_line.gross.amount.quantize(Decimal("0.01"))
-        ),
-        "tax_rate": str(line.tax_rate.quantize(Decimal("0.0001"))),
-        "charge_taxes": line.variant.product.charge_taxes,
-        "allocations": [
-            {
-                "warehouse_id": global_warehouse_id,
-                "quantity_allocated": allocation.quantity_allocated,
-            }
-        ],
-        "undiscounted_unit_price_net_amount": str(undiscounted_unit_price_net_amount),
-        "undiscounted_unit_price_gross_amount": str(
-            undiscounted_unit_price_gross_amount
-        ),
-        "undiscounted_total_price_net_amount": str(undiscounted_total_price_net_amount),
-        "undiscounted_total_price_gross_amount": str(
-            undiscounted_total_price_gross_amount
-        ),
-        "product_metadata": product.metadata,
-        "product_type_metadata": product_type.metadata,
-        "voucher_code": line.voucher_code,
-        "sale_id": line.sale_id,
-    }
+    assert line_payload["id"] == line_id
```

Comment:
The previous assert was checking that payload doesn't have any additional fields. Now you can check 10 fields where `line_payload` will have 20 fields.

Reply:
Resolved in recent commit.